README.md

```diff
@@ -112,6 +112,33 @@ Now you can compile your SASS files running:
 ```
 Which will take all the `scss` files inside the `styles/` folder and create the `style.css` file that is consumable by Zendesk Guide.
 
+
+# Quick Setup Recall (Running and testing theme locally)
+
+NOTE: This guide is to quickly setup and run the theme locally. To know more about how to customize the theme, See all the details above.
+
+1. Create a trial account on https://www.zendesk.com/. This will let you create a subdomain Zendesk account that you can manage.
+2. Make sure to opt-in for Guide integration once your trial account setup is done.
+3. Create API keys for running the server locally and previewing them on your subdomain. `<YOUR_ZENDESK_DOMAIN>/admin/apps-integrations/apis/zendesk-api/settings`.
+4. Enable Help Center for your account through `<YOUR_ZENDESK_DOMAIN>/hc/admin/general_settings`
+5. Install [ZAT](https://developer.zendesk.com/documentation/apps/zendesk-app-tools-zat/installing-and-using-zat/) and [ZAT CLI](https://support.zendesk.com/hc/en-us/articles/4408822095642-Previewing-theme-changes-locally)
+6. Run `npm i` to install all the node packages
+7. Install sassc as mentioned in [Requirements sections](#requirements)
+8. Create a `.zat` file in your main directory. You can use `zat new` or create one manually. This file content would look like:
```

Comment:
`zat new` creates a new project. No? I thought that Ibrahim mentioned it while he was working on the ocw-zendesk changes.

Reply:
Yes, I'm removing that step.

Reply:
I've added a reference to https://developer.zendesk.com/documentation/apps/zendesk-app-tools-zat/installing-and-using-zat/#configuring-updates in the README instead.